src/components/TagSelect/index.tsx

```diff
@@ -60,21 +61,9 @@ const TagSelect = ({ options = [], onChange = () => {}, selected, ...props }: IT
           </li>
         )}
       </S.Dropdown>
-      <Arrow color={isOpen ? '#141414' : selected.color} />
+      <Arrow direction="down" size="small" fill={isOpen ? '#141414' : selected.color} />
```

Comment:
If this is a non-brand colour can you add a TODO comment here?

Reply:
Brand color added!